RecoMET/METAlgorithms/src/METSignificance.cc

```diff
@@ -90,17 +85,13 @@ metsig::METSignificance::getCovariance(const edm::View<reco::Jet>& jets,
       double jpt  = jet->pt();
       double jeta = jet->eta();
       double feta = std::abs(jeta);
-      double c = std::cos(jet->phi());
-      double s = std::sin(jet->phi());
+      double c = jet->px()/jet->pt();
```

Comment:
I missed that one, will be corrected with the output format once https://github.com/cms-sw/cmssw/pull/7586 is merged